page/jquery-ui/themeroller.md

```diff
@@ -36,13 +34,4 @@ Once the theme has been downloaded and unzipped, you will see a folder named `th
 
 ThemeRoller generates a customized version of the jQuery UI CSS Framework for developing your own ThemeRoller-ready jQuery components. The classes generated by this framework are designed to accommodate common user interface design situations and include states, icons, and various helper classes as well.
 
-For information on developing with the jQuery UI CSS Framework, visit our [Theming API documentation](/jquery-ui/theming/api).
-
-### ThemeRoller Links
-* [Tutorial: Develop Your Own jQuery "ThemeRoller-Ready" Components, Filament Group](http://www.filamentgroup.com/lab/developer_your_own_jquery_themeroller_ready_components/)
-* [Introducing ThemeRoller: Design & Download Custom Themes for jQuery UI, Filament Group](http://www.filamentgroup.com/lab/introducing_themeroller_design_download_custom_themes_for_jquery_ui/)
-* [Videos about Design & Download Custom Themes for jQuery UI, Filament Group](http://vimeo.com/1113981?pg=embed&sec=1113981)
-
-### Credits
```

Comment:
That applied to the initial version, the current version was almost completely rewritten.